BESMod/Systems/Hydraulical/Distribution/Types/EnergyLabel.mo

```diff
@@ -0,0 +1,11 @@
+within BESMod.Systems.Hydraulical.Distribution.Types;
+type EnergyLabel = enumeration(
+    labelAPlus "A+",
+    labelA "A",
+    labelB "B",
+    labelC "C",
+    labelD "D",
+    labelE "E",
+    labelF "F",
+    labelG "G")
```

Comment:
Is it necessary to have `label` in front of every option? I think `APlus "A+",` would also be fine, as we always will use it like this: `BESMod.Systems.Hydraulical.Distribution.Types.EnergyLabel.A`